### ISSUE TITLE
Issue: Timed out after waiting {time} ms for your remote page to load on Electron and Chrome Browser

### ISSUE THREAD

**Issue by ankitag06** [cypress-io/cypress#7929]
### Current behavior: 

When we click on Log Out button or any button in our application, Instead of navigating on other page, it throws Page Load Time Out error. I have tried using explicit wait `cy.wait(5000)` or increased Page Load Time out but nothing is working with Browser Chrome and Electron

![image](https://user-images.githubusercontent.com/68022700/87033338-f7f23400-c1dd-11ea-8209-981ed3ba8d9a.png)

Hi @jennifer-shehane,

Here is the new ticket for the issue which I  mentioned on ticket: Timed out after waiting {time} ms for your remote page to load occurring only in headless mode (#7628)

### Desired behavior: 

There is a scenario in our application to Schedule Transition if we try click on submit button, it should navigate on Scheduled transition page  or if we click on log out button , it should log out and navigate on Log in Page.

![tempsnip](https://user-images.githubusercontent.com/68022700/87033523-48699180-c1de-11ea-841e-1f1713d77823.png)

### Test code to reproduce

Sorry, I can't give much details of the application since it will violate the Organization's policy.
Just to let you know,this is working on Firefox Browser

### Versions

Cypress: 4.9.0
Browser: Chrome, Electron
Operation system: windows 64 bit

**Comment by jennifer-shehane** [comment]
Hmm, there's not much to investigate here without a reproducible example. 

- Are there any errors in the DevTools console?
- Do you have any listeners/code that would run on an `onbeforeunload` or similar that may be being hit?
- Can you provide the screenshot of the Firefox run?

Overall, we'll need a way to see the error ourselves in order to investigate why this is happening though.

**Comment by ankitag06** [comment]
Hi @jennifer-shehane 

This is working on Firefox browser without any explicit wait or increased Page load timeout
![image](https://user-images.githubusercontent.com/68022700/87137367-aeb0eb80-c294-11ea-833d-e3e038aca79e.png)

1. Yes, there is 1 error in console 
2. No,we don't have any listeners/code that would run on an onbeforeunload
3. yes, Please find below screenshots:

Electron Browser - Console:
![image](https://user-images.githubusercontent.com/68022700/87138291-19aef200-c296-11ea-822b-fd58bc918996.png)

Chrome Browser - Console: 
![image](https://user-images.githubusercontent.com/68022700/87138718-c12c2480-c296-11ea-80c0-5df1bba1a567.png)

**Comment by smaurya07** [comment]
Hi @jennifer-shehane 

I am facing similar issue with [https://www.screener.in/company/INFY/consolidated/](https://www.screener.in/company/INFY/consolidated/) when trying to download an excel file by clicking on 'Export to excel' button.

This fails for chrome, firefox and electron also. You could use this example to debug the issue. This is public website and you can download file only after login.

<img width="1679" alt="Screenshot 2020-07-19 at 9 44 28 PM" src="https://user-images.githubusercontent.com/39452636/87879638-cfeaa800-ca09-11ea-9a8b-b88f96ab8b9f.png">

**Comment by ab3llo** [comment]
Any update on this, I'm getting a similar issue for a test that creates a user via api, uses the data from api response to the fill the form in and log in. The login page is displayed and loaded but cypress pageLoad hangs until it timeout

**Comment by srodrigo101ways** [comment]
I'm having the same issue when submitting a form. No `window.onbeforeunload` function set.

This looks like a pretty common issue.

**Comment by prashantabellad** [comment]
@jennifer-shehane @bahmutov is there any resolution for this?

**Comment by bahmutov** [comment]
No updates, since we do not have a reproducible example. I would suggest looking at our file download and network stubbing recipes in https://github.com/cypress-io/cypress-example-recipes

**Comment by jennifer-shehane** [comment]
Right now there doesn't seem to be enough information to reproduce the problem on our end. We'll have to close this issue until we can reproduce it. This does not mean that your issue is not happening - it just means that we do not have a path to move forward. 

Please comment in this issue with a reproducible example and we will consider reopening the issue. Here are some tips for providing a [Short, Self Contained, Correct, Example](http://sscce.org/) and our own [Troubleshooting Cypress](https://on.cypress.io/troubleshooting) guide.
